tutorials/continuous-aggs-tutorial.md

```diff
@@ -0,0 +1,238 @@
+```
+The refresh_interval is set to 30 minutes. The computed aggregates are saved 
+in the materialization table,  _timescaledb_internal._materialized_hypertable_2. 
+
+What is refresh_lag and max_interval_per_job? We use the 
+`timescaledb.refresh_lag` parameter to indicate by how much does the continuous
+ aggregate query lag behind the data in the *rides* table. For example, if we 
+expect frequent updates to the *rides*  table for the current hour, we do not 
+want to precompute the aggregates for that range. We would set the 
+**refresh_lag = ‘1h’** to indicate that. (The default value is twice the 
+bucket_width used by the time_bucket expression. This is the 2 hours shown for 
+refresh_lag in the view output above). So the continuous aggregate will get 
+refreshed every 30 minutes (refresh_interval) but will update the continuous 
+aggregates only for the data that satisfies the condition: 
+time_bucket(‘1h’, pickup_datetime) <   max(pickup_time) - ‘1h’  (if the 
+refresh_lag is set to 1 hour)
+
+The `timescaledb.max_interval_per_job` parameter is used when we want to limit 
+the amount of data processed by an update of the continuous aggregate and use 
+smaller or bigger batch sizes (the batching is done automatically by TimescaleDB). 
+`timescaledb.`max_interval_per_job` specifies the batch size.
+
+refresh_lag and max_interval_per_job are additional parameters that can be 
+specified while creating or altering a continuous aggregate. Refer to the 
+documentation for the syntax.
+
+### 3. Queries using continuous aggregates
+We can use the continuous aggregate, just like any other view, in a SELECT query.
+
+``` sql
+SELECT vendor_id, day, total_rides FROM cagg_rides_view WHERE total_rides > 15000;
+ vendor_id |         day         | total_rides 
+-----------+---------------------+-------------
+ 2         | 2016-01-01 01:00:00 |       15407
+(1 row)
+```
+
+### 4. Statistics for continuous aggregates
+
+We can view information about the jobs that updated the continuous aggregates 
+using the **timescaledb_information.continuous_aggregate_stats** view.
+
+``` sql
+SELECT * FROM timescaledb_information.continuous_aggregate_stats;
+
+-[ RECORD 1 ]----------+------------------------------
+view_name              | cagg_rides_view
+completed_threshold    | 2016-01-31 22:00:00
+invalidation_threshold | 2016-01-31 22:00:00
+job_id                 | 1000
+last_run_started_at    | 2019-04-25 10:48:08.15141-04
+job_status             | scheduled
+last_run_duration      | 00:00:00.042841
+next_scheduled_run     | 2019-04-25 11:18:08.194251-04
+
+SELECT max(pickup_datetime) FROM rides;
+-[ RECORD 1 ]------------
+max | 2016-01-31 23:59:59
+
+```
+
+Job_id gives the id of the background worker that updates the continuous 
+aggregate query. next_scheduled_run says when is the next scheduled update. 
+The completed_threshold shows that rows with 
+"pickup_time" value < '2016-01-31 22:00:00' (from the *rides* table) were used 
```

Comment:
`pickup_time` in backticks.  Also, since you are describing a threshold within a paragraph, maybe use the text "less than" rather than the symbol "<"?

Reply:
I prefer using  < symbol instead of the text.